src/lib.rs

```diff
@@ -68,11 +67,11 @@ pub fn parse_module(
   maybe_kind: Option<ModuleKind>,
   maybe_resolver: Option<&dyn Resolver>,
   maybe_module_analyzer: Option<&dyn ModuleAnalyzer>,
-) -> Result<Module, ModuleGraphError> {
+) -> Result<Module, Box<ModuleGraphError>> {
```

Comment:
Clippy told me to do this, so I listened... I don't get it to be honest though.

Reply:
Here's the rationale for the lint: https://rust-lang.github.io/rust-clippy/master/index.html#result_large_err

I feel like the `Ok` variant will always be larger though?

Reply:
Seems reasonable, though I never hit it myself.